examples/util/openvino/OpenVINOModelImporter.js

```diff
@@ -681,6 +737,90 @@ class OpenVINOModelImporter {
 
           opCode = this._nn.SOFTMAX;
         } break;
+        case 'Normalize': {
+          const input = node.inputs[0];
+          const inDims = input.shape();
+          const inputId = this._getTensorId(input);
+          console.log(`  input shape: [${input.shape()}]`);
+
+          const acrossSpatial = node.getInt('across_spatial');
+          const channelShared = node.getInt('channel_shared');
+          const eps = node.getFloat('eps');
+
+          if (acrossSpatial !== 0 || channelShared !== 0) {
+            throw new Error(`Normalize not support across_spatial ${across_spatial} channel_shared ${channel_shared}.`);
+          }
+
+          inputs.push(inputId);
+
+          const output = node.outputs[0];
+
+          if (i === graph.nodes.length - 1) {
+            this._tensorIds[output.graphId()] = inputId;
+            console.log(`  skip last op: Normalize`);
+          } else {
+            const outDims = output.shape();
+            const outputType = {
+              type: this._getTypeCode(output.dataType()), dimensions: outDims
+            };
+            const outputId = this._addNamedOperand(output.graphId(), outputType);
+            outputs.push(outputId);
+            console.log(`  output shape: [${outDims}]`);
+
+            opCode = this._nn.L2_NORMALIZATION;
+          }
+        } break;
+        case 'PReLU': {
+          const input = node.inputs[0];
+          const inDims = input.shape();
+          console.log(`  input shape: [${input.shape()}]`);
+
+          const slope = node.inputs[1];
+          const slopeTensor = slope.getInitializer();
+          const slopeDims = [slopeTensor.length];
+          console.log(`  slope shape: [${slopeDims}]`);
+
+          inputs.push(this._getTensorId(input));
+          inputs.push(this._addTensorFloat32(slopeTensor, slopeDims));
+
+          const output = node.outputs[0];
+          const outDims = output.shape();
+          const outputType = {
+            type: this._getTypeCode(output.dataType()), dimensions: outDims
+          };
+          const outputId = this._addNamedOperand(output.graphId(), outputType);
+          outputs.push(outputId);
+          console.log(`  output shape: [${outDims}]`);
+
+          opCode = this._nn.PRELU;
+        } break;
+        case 'Activation': {
+          // Add inputs
+          const in1 = node.inputs[0];
+          inputs.push(this._getTensorId(in1));
+          console.log(`  inputs shape: ` +
+              `[${node.inputs.map((input) => input.shape()).join('], [')}]`);
+
+          const operation = node.getString('type');
+          console.log(`  operation: ${operation}`);
+          switch (operation) {
+            case 'sigmoid':
+              opCode = this._nn.LOGISTIC;
+              break;
+            default:
+              throw new Error(`Operation ${operation} is not supported`);
```

Comment:
Better to show error message as "The type ${type} of Activation is not supported"